tests/integration/defs/accuracy/test_llm_api.py

```
@@ -76,6 +78,27 @@ def test_guided_decoding_4gpus(self, backend: str):
             task = JsonModeEval(self.MODEL_NAME)
             task.evaluate(llm)
 
+    def test_gather_generation_logits_cuda_graph(self):
+        "RCCA: https://nvbugs/5365525"
```

Ruff (0.12.2)

First line should end with a period, question mark, or exclamation point

Add closing punctuation

(D415)